### ISSUE TITLE
client.devices.interfaces.get_all(device.id) fails due to top_nexthop_asns parsing

### ISSUE THREAD

**Issue by dpajin-kentik** [kentik/community_sdk_python#83]
I was using the function to retrieve interfaces information from the device with the following InterfaceAPI function: `client.devices.interfaces.get_all(device.id)`.

I am getting the error below. 

I think this is caused by some interfaces which are returned from API with the empty list for `top_nexthop_asns` instead of `None`, for example in the EU cluster, kentik organization:
device name: edge02_fra1
        "device_id": "1176",
        "snmp_id": "519",
```
"top_nexthop_asns": [],
```
There are also other outputs like: 
```
"top_nexthop_asns": null,
```
or 
```
        "top_nexthop_asns": [
            {
                "Asn": 174,
                "Packets": 42826176
            }
        ],

        "top_nexthop_asns": [
            {
                "Asn": 15169,
                "Packets": 6167872
            },
            {
                "Asn": 59605,
                "Packets": 1698400
            },
            {
                "Asn": 42708,
                "Packets": 750496
            },
            {
                "Asn": 25091,
                "Packets": 201376
            },
            {
                "Asn": 24482,
                "Packets": 29600
            },
            {
                "Asn": 39326,
                "Packets": 22624
            },
            {
                "Asn": 9009,
                "Packets": 9856
            },
            {
                "Asn": 61098,
                "Packets": 6592
            }
        ],
```

I am using kentik-api  v0.2.0. 

The traceback output:
```
Traceback (most recent call last):
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/dacite/core.py", line 59, in from_dict
    field_data = data[field.name]
KeyError: 'ASN'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/dacite/core.py", line 71, in from_dict
    value = get_default_value_for_field(field)
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/dacite/dataclasses.py", line 21, in get_default_value_for_field
    raise DefaultValueNotFoundError()
dacite.dataclasses.DefaultValueNotFoundError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/conversions.py", line 56, in from_dict
    return dacite.from_dict(data_class=data_class, data=data)
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/dacite/core.py", line 75, in from_dict
    raise MissingValueError(field.name)
dacite.exceptions.MissingValueError: missing value for field "ASN"

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/conversions.py", line 105, in convert
    return convert_func(attr)
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/interfaces_payload.py", line 47, in from_dict
    return from_dict(data_class=cls, data=dic)
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/conversions.py", line 58, in from_dict
    raise DeserializationError(data_class.__name__, str(err)) from err
kentik_api.public.errors.DeserializationError: TopNextHopASNPayload: missing value for field "ASN"

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "api_test.py", line 60, in <module>
    intfs = client.devices.interfaces.get_all(device.id)
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/api_resources/devices_api.py", line 17, in get_all
    return interfaces_payload.GetAllResponse.from_json(response.text).to_interfaces()
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/interfaces_payload.py", line 232, in from_json
    interfaces = [GetResponse.from_dict(item) for item in items]
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/interfaces_payload.py", line 232, in <listcomp>
    interfaces = [GetResponse.from_dict(item) for item in items]
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/interfaces_payload.py", line 219, in from_dict
    return cls(interface=InterfacePayload.from_dict(dic))
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/interfaces_payload.py", line 164, in from_dict
    top_nexthop_asns=convert_list_or_none(dic.get("top_nexthop_asns"), TopNextHopASNPayload.from_dict),
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/conversions.py", line 123, in convert_list_or_none
    return [convert(item, convert_func) for item in items]
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/conversions.py", line 123, in <listcomp>
    return [convert(item, convert_func) for item in items]
  File "/home/dpajin/.venv/kentik-api/lib/python3.8/site-packages/kentik_api/requests_payload/conversions.py", line 107, in convert
    raise DataFormatError(str(err)) from err
kentik_api.public.errors.DataFormatError: TopNextHopASNPayload: missing value for field "ASN"
```

**Comment by mmac-m3a** [comment]
Dušan, thanks for the report. This problem has actually been fixed in the code some time ago, but the code has not been released. It will be released soon (along with a bigger change). If you need the problem fixed now, please, check out the code from the repo and install it locally. Please, let me know if you need help with that.

**Comment by dpajin-kentik** [comment]
Thanks @mmac-m3a, I just tested with the code from the main branch and it works. Closing the issue.

**Comment by mmac-m3a** [comment]
Fix released in v0.2.1

**Comment by dpajin-kentik** [comment]
tested again with v0.2.1 and it works
